sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -36,6 +36,17 @@ Use FarmBeats client library for Python to do the following.
 
 In order to interact with the Azure FarmBeats service, your client must present an Azure Active Directory bearer token to the service.
 
+After setup, you can choose which type of [credential][azure_identity_credential_type] from azure-identity to use.
+We recommend using [DefaultAzureCredential][wiki_identity], configured through the AZURE_TOKEN_CREDENTIALS environment variable.
+
+Set this variable to one of the following groups:
+
+- dev: for local development credentials (CLI, IDE, etc.)
+
+- prod: for production-ready credentials (managed identity, workload identity, etc.)
+
```

Comment:
I recommend removing this content and directing customers to the Learn doc instead. We want that doc to be the source of truth.

```suggestion
```